pyccel/codegen/python_wrapper.py

```diff
@@ -144,9 +145,11 @@ def create_shared_library(codegen,
                 verbose=verbose)
         timings['Bind C wrapping'] = time.time() - start_bind_c_compiling
         c_ast = bind_c_mod
+    elif language == 'cuda':
+        wrapper = CudaToCWrapper()
+        c_ast = wrapper.wrap(codegen.ast)
     else:
         c_ast = codegen.ast
-
     #---------------------------------------
```

Comment:
Bad vspace change. I think this vspace helps with the readability

Reply:
fixed https://github.com/pyccel/pyccel-cuda/commit/e58eddfc457591d14aa98876d4f21044debc7e2e